rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -450,3 +450,41 @@ def _list_oflow_count_for_sandboxes(self, sandboxes,
             "chart_plugin": "StackedArea", "data": oflow_data
         }
         self.add_output(additive_oflow_data)
+
+    def _create_address_set(self, set_name, address_list):
+        LOG.info("create %s address_set [%s]" % (set_name, address_list))
+
+        name = "name=\"" + set_name + "\""
+        addr_list="\"" + address_list + "\""
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.create("Address_Set", name, ('addresses', addr_list))
+        ovn_nbctl.flush()
+
+    def _modify_address_set(self, set_name, address_list):
```

Comment:
The name suggests the operation could be "add" or "delete" addresses to/from the address set, but the implementation is just "add". Please either implement both operations (the parameter can be addresses_to_add, addresses_to_delete), or rename the function to something like _address_set_add_addrs